examples/controlnet-interior-design/components/download_images/fondant_component.yaml

```diff
@@ -0,0 +1,48 @@
+name: Download images
+description: Component that downloads actual images based on URLs
+image: download_images:latest
+
+input_subsets:
+  images:
+    fields:
+      url:
+        type: utf8
+
+output_subsets:
+  images:
+    fields:
+      data:
+        type: binary
+      width:
+        type: int
+      height:
+        type: int
+      original_width:
+        type: int
+      original_height:
+        type: int
+      error_message:
+        type: utf8
```

Comment:
Seems strange to me to output an error_message. Would drop images with errors instead.